Add notes about double forging - Closes #403 (#433)
LiskHQ/lisk-docs: modules/ROOT/pages/management/forging.adoc

----
$ lisk
lisk passphrase:encrypt --output-public-key
Please enter your secret passphrase: ***** <1>
Please re-enter your secret passphrase: *****
Please enter your password: *** <2>
Please re-enter your password: ***
{
        "encryptedPassphrase": "iterations=1000000&cipherText=30a3c8&iv=b0d7322bf24e0dfe08462f4f&salt=aa7e26c9f4317b61b4f45b5c6909f941&tag=a2e0eadaf1f11a10b342965bc3bafc68&version=1",
        "publicKey": "a4465fd76c16fcc458448076372abf1912cc5b150663a64dffefe550f96feadd"
}
----

> <1> Enter here the secret passphrase that needs to be encrypted.

Reviewer: ```suggestion
<1> Enter the secret passphrase here that needs to be encrypted.
```

> <2> Enter here the password that will be required to decrypt the passphrase again.

Reviewer: ```suggestion
<2> Enter the password here that will be required to decrypt the passphrase again.
```

> * Type in your passphrase followed by the password required for encryption.

Reviewer: ```suggestion
* Type in the passphrase followed by the password required for encryption.
```

* This will result in the creation of an `encryptedPassphrase` key-value pair.

> * Create the JSON object and add it to your `config.json` under `chain.forging.delegates` as can be seen below:

Reviewer: ```suggestion
* Create the JSON object and add it to the `config.json` under `chain.forging.delegates` as can be seen below:
```

<5> Contains options for the API module.

> <6> Replace with the IP which will be used to access your node

Reviewer: ```suggestion
<6> Replace with the IP address which will be used to access the node
```

> Reload your Lisk Core process to make the changes in the config effective.

Reviewer: ```suggestion
Reload the Lisk Core process to perform the changes in the config effective.
```

> For example, for Binary install, run the following command: `bash lisk.sh reload`

Reviewer: ```suggestion
For example, for Binary install, execute the following command: `bash lisk.sh reload`
```

> IMPORTANT: Remember that after restarting your Lisk node, it is necessary to re-enable forging again.

Reviewer: ```suggestion
IMPORTANT: Remember that after restarting the Lisk node, it is necessary to re-enable forging again.
```

IMPORTANT: Remember that after restarting your Lisk node, it is necessary to re-enable forging again.

CAUTION: Do not activate forging for the same delegate on multiple nodes.
This behaviour will be punished by the network.

> For more info, go to the xref:{url_sdk_best_practices}[Lisk SDK - Best ptractices] page.

Reviewer: ```suggestion
For more information, please see xref:{url_sdk_best_practices}[Lisk SDK - Best practices] page.
```
